pcdsdevices/mpodApalis.py

```diff
@@ -0,0 +1,288 @@
+import logging
```

Comment:
The name of this file violates pep8: https://www.python.org/dev/peps/pep-0008/#package-and-module-names